### PR TITLE
Add documentation for using vagrant

### PR DESCRIPTION
**Pull request by d0**
Add a bit of documentation for using vagrant to setup a development or testing environment. Also add a symlink to the shared folder in the vagrant environment.

### CODE REVIEW

File: doc/install.txt.in [hunk 1]
```diff
@@ -66,6 +66,20 @@ You need to explicitely configure @PACKAGE_NAME@ to install them by using
 :option:`--enable-python`, :option:`--enable-java`, ... This requires SWIG to
 be installed along with the language's toolchain to build the bindings.
 
+==========================================================
+Setting up a development or test environment using vagrant
+==========================================================
+
+The easiest way to setup a development or test environment is to use vagrant.
```
Comment (frankmorgner) [inline doc/install.txt.in]: Could you add a link to the vagrant website?

**Comment by frankmorgner** [comment]
nice!

File: doc/install.txt.in (hunk 1 above)
Comment (d0) [inline doc/install.txt.in]: Added in c8400697e97c7fe973dddc26fbe4d2aac7d8625e

**Comment by frankmorgner** [comment]
thanks

**Pull request merged**